triton-vm/src/table/base_matrix.rs

```diff
@@ -173,6 +178,75 @@ impl BaseMatrices {
         {
             ram_matrix[idx][usize::from(RamBaseTableColumn::InverseOfRampDifference)] = inverse;
         }
+
+        // compute Bézout coefficient polynomials
+        let all_ramps = ram_matrix
+            .iter()
+            .map(|row| row[usize::from(RAMP)])
+            .dedup()
+            .collect_vec();
+        let num_of_different_ramps = all_ramps.len();
+        let polynomial_with_ramps_as_roots = all_ramps
+            .into_iter()
+            .map(|ramp| Polynomial::new(vec![-ramp, BFieldElement::one()]))
+            .fold(
+                Polynomial::from_constant(BFieldElement::one()),
+                Polynomial::mul,
+            );
+
+        let formal_derivative_coefficients = polynomial_with_ramps_as_roots
+            .coefficients
+            .iter()
+            .enumerate()
+            .map(|(i, &coefficient)| BFieldElement::new(i as u64) * coefficient)
+            .skip(1)
+            .collect_vec();
+        let formal_derivative = Polynomial::new(formal_derivative_coefficients);
+
+        let (gcd, bezout_0, bezout_1) =
+            XFieldElement::xgcd(polynomial_with_ramps_as_roots, formal_derivative);
```

Comment:
Yes, this method currently lives in `XFieldElement`. A petition to move it to `Polynomial` has been approved and is currently being implemented.

The same holds true for a method `formal_derivative`.